preload_associations.go

```diff
@@ -491,6 +492,12 @@ func preloadManyToMany(tx *Connection, asoc *AssociationMetaInfo, mmi *ModelMeta
 			fkids = append(fkids, row[1])
 		}
 	}
+	if err := rows.Close(); err != nil {
```

Comment:
This `Close()` is not strictly necessary since we iterate over `rows.Next()`, but making it explicit is probably not a bad thing.

Reply:
no harm but explicitly redundant.